.github/workflows/alternative-ci-runners-3.yml

```diff
@@ -0,0 +1,45 @@
+name: Alternative CI Runners 3
+
+on:
+  # Run the workflow every day TWICE:
+  # 1. 9:06AM UTC (low activity)
+  # 2. 9:26AM UTC (cache test - high chance of no code changes)
+  schedule:
+    - cron: "6,26 9 * * *"
+  # Enable manual trigger for on-demand runs - helps when debugging
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  docs-lint-on-magicache:
+    uses: ./.github/workflows/_dagger_on_magicache.yml
+    with:
+      function: docs lint
+      timeout: 20
+
+  test-cli-engine-on-magicache:
+    needs: docs-lint-on-magicache
+    uses: ./.github/workflows/_dagger_on_magicache.yml
+    with:
+      function: test specific --run='TestCLI|TestEngine' --race=true --parallel=16
+      timeout: 20
```

Comment:
`test-cli-engine` is most resource intensive. It assumes & uses 16 CPUs. FTR:
<img width="1520" alt="image" src="https://github.com/user-attachments/assets/bd9f53de-e594-4187-95ac-8cf6ae00b017" />

We should choose one of the following options:
- run on 16CPUs
- remove it

Reply:
Fixed by running on 16c runners **only** - same as all other alternative workflows.